bears/general/QuotesBear.py

```diff
@@ -34,9 +35,16 @@ def correct_single_line_str(self, filename, file, sourcerange,
         str_contents = file[sourcerange.start.line - 1][
                        sourcerange.start.column:sourcerange.end.column-1]
 
-        if preferred_quotation in str_contents:
+        if (preferred_quotation in str_contents and
+                not force_preferred_quotation):
             return
 
+        elif preferred_quotation in str_contents:
+            str_split = str_contents.split(preferred_quotation)
+            for i in range(len(str_split) - 1):
+                str_split[i] += '\\'
+            str_contents = preferred_quotation.join(str_split)
```

Comment:
maybe use `str.replace`? Then you don't need the `if` probably, just replace any of those characters if available :+1:

Reply:
Yeah str.replace can be used, but then also the `if` is to be used.